repositories.bzl

```diff
@@ -185,6 +185,53 @@ filegroup(
             actual = "@mixerapi_git//:mixer_client_config_cc_proto",
         )
 
+def authenticationapi_repositories(bind=True):
+    BUILD = """
+# Copyright 2018 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
+
+cc_proto_library(
```

Comment:
just add this into line 164, use one git_reposity for api repo

Reply:
Done.

Reply:
Done.